dataplicity/m2m/commandservice.py

```diff
@@ -38,54 +38,83 @@ def __init__(self, channel, command):
     def __repr__(self):
         return self._repr
 
+    @classmethod
+    def send_error(cls, channel, status, msg, **extra):
```

Comment:
Looks like this is only being called on instances. Would this work if it wasn't a `classmethod`?

Also, `extra` doesn't appear to be invoked, might that be able to go too?